website/docs/cdktf/concepts/iterators.mdx

```diff
@@ -98,29 +177,29 @@ new TeamMembers(this, "members", {
 ```
 
 ```python
-org_name = "my-org"
+        org_name = "my-org"
 
-GithubProvider(self, "github",
-    organization=org_name
-)
+        github.provider.GithubProvider(self, "github",
+                                       organization=org_name
+                                       )
 
-team = Team(self, "core-team",
-    name="core"
-)
+        github_team = github.team.Team(self, "core-team",
+                                       name="core"
+                                       )
 
-org_members = DataGithubOrganization(self, "org",
-    name=org_name
-)
+        org_members = github.data_github_organization.DataGithubOrganization(self, "org",
+                                                                             name=org_name
+                                                                             )
 
-org_member_iterator = TerraformIterator.from_list(org_members.members)
+        org_member_iterator = TerraformIterator.from_list(org_members.members)
 
-github.TeamMembers(self, "members",
-    team_id=team.id,
-    members=org_member_iterator.dynamic({
-        "username": Token().as_string(org_member_iterator.value),
-        "role": "maintainer"
-    })
-)
+        github.team_members.TeamMembers(self, "members",
+                                        team_id=github_team.id,
+                                        members=org_member_iterator.dynamic({
+                                            "username": Token().as_string(org_member_iterator.value),
+                                            "role": "maintainer"
+                                        })
+                                        )
```

Comment:
As commented above, we might want to get rid of all this indentation in a follow-up PR.
As opposed to the other comment, for this sample it would be enough to remove all leading indentation that all lines share.
The script should already do that [here](https://github.com/hashicorp/terraform-cdk/blob/39241c4640da6116485d8fb6ea243f596080dc36/tools/documentation-code-snippets/update-documentation-code-snippets.ts#L222), but that logic seems to have a bug (it should ignore empty lines, as they currently set the min value to 0 😅)

Reply:
👍